Tools/ardupilotwaf/cxx_checks.py

```diff
@@ -157,26 +157,46 @@ def check_librt(cfg, env):
     return ret
 
 @conf
+def check_package(cfg, env, libname):
+    '''use pkg-config to look for an installed library that has a LIBNAME.pc file'''
+    cfg.check_cfg(package=libname, mandatory=False, global_define=True,
+                  args=['--libs', '--cflags'])
```

Comment:
I have tried a number of varients of uselib_store, and have yet to find one that correctly adds the include, lib and cflags vars. Can you suggest a full check_cfg() line to try?